github/orgs_audit_log.go

```diff
@@ -0,0 +1,105 @@
+	Actor                *string    `json:"actor,omitempty"`
+	BlockedUser          *string    `json:"blocked_user,omitempty"`
+	Business             *string    `json:"business,omitempty"`
+	CancelledAt          *Timestamp `json:"cancelled_at,omitempty"`
+	CompletedAt          *Timestamp `json:"completed_at,omitempty"`
+	Conclusion           *string    `json:"conclusion,omitempty"`
+	Config               *string    `json:"config,omitempty"`
+	ConfigWas            *string    `json:"config_was,omitempty"`
+	ContentType          *string    `json:"content_type,omitempty"`
+	CreatedAt            *int64     `json:"created_at,omitempty"`
+	DeployKeyFingerprint *string    `json:"deploy_key_fingerprint,omitempty"`
+	DocumentID           *string    `json:"_document_id,omitempty"`
+	Emoji                *string    `json:"emoji,omitempty"`
+	EnvironmentName      *string    `json:"environment_name,omitempty"`
+	Event                *string    `json:"event,omitempty"`
+	Events               *string    `json:"events,omitempty"`
+	EventsWere           *string    `json:"events_were,omitempty"`
+	Explanation          *string    `json:"explanation,omitempty"`
+	Fingerprint          *string    `json:"fingerprint,omitempty"`
+	HeadBranch           *string    `json:"head_branch,omitempty"`
+	HeadSHA              *string    `json:"head_sha,omitempty"`
+	HookID               *string    `json:"hook_id,omitempty"`
+	IsHostedRunner       *bool      `json:"is_hosted_runner,omitempty"`
+	JobName              *string    `json:"job_name,omitempty"`
+	LimitedAvailability  *string    `json:"limited_availability,omitempty"`
+	Message              *string    `json:"message,omitempty"`
+	Name                 *string    `json:"name,omitempty"`
+	OldUser              *string    `json:"old_user,omitempty"`
+	OpenSSHPublicKey     *string    `json:"openssh_public_key,omitempty"`
+	Org                  *string    `json:"org,omitempty"`
+	PreviousVisibility   *string    `json:"previous_visibility,omitempty"`
+	ReadOnly             *string    `json:"read_only,omitempty"`
+	Repo                 *string    `json:"repo,omitempty"`
+	Repository           *string    `json:"repository,omitempty"`
+	RepositoryPublic     *bool      `json:"repository_public,omitempty"`
+	RunnerGroupID        *string    `json:"runner_group_id,omitempty"`
+	RunnerGroupName      *string    `json:"runner_group_name,omitempty"`
+	RunnerID             *string    `json:"runner_id,omitempty"`
+	RunnerLabels         *[]string  `json:"runner_labels,omitempty"`
+	RunnerName           *string    `json:"runner_name,omitempty"`
+	SecretsPassed        *[]string  `json:"secrets_passed,omitempty"`
+	SourceVersion        *string    `json:"source_version,omitempty"`
+	StartedAt            *Timestamp `json:"started_at,omitempty"`
+	TargetLogin          *string    `json:"target_login,omitempty"`
+	TargetVersion        *string    `json:"target_version,omitempty"`
+	Team                 *string    `json:"team,omitempty"`
```

Comment:
A `Team` doesn't point to a `*Team`?  It's really just a string?

Reply:
that's correct, it's just a string of the team name. Since the Audit Log is mostly immutable logs, we don't store full object details (in the event that the team / repo / user account has been deleted for example).